server/backend/api/app/crud/crud_application.py

```diff
@@ -119,6 +119,35 @@ def getFamApplicationRoles(
     )
     return application
```

Comment:
Hi Kevin, just notice this, probably is a variable name change missed, I guess it should be 'roles'?

Reply:
Thanks for that @ianliuwk1019, resolved now